### PR TITLE
db table creation modified for RemoteFetch Configuration

### PR DESCRIPTION
**Pull request by Thumimku**
## Purpose
The current implementation of RemoteFetchConfiguration using the auto-increment id for both remote fetch configuration and deployment revision. The purpose of this PR is to change the auto-increment id to UUID, not only in database tables, but also in java level. 

## Related PRs
https://github.com/wso2-extensions/identity-fetch-remote/pull/25

## Automation tests
 - Unit tests 
   >passed all unit tests

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

- H2
- MSSQL
- MySQL
- Oracle 12c
- DB2
- Postgres

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql [hunk 1]
```diff
@@ -1533,7 +1533,7 @@ IDN_CONFIG_RESOURCE (ID) ON DELETE CASCADE ON UPDATE RESTRICT
 /
 
 CREATE TABLE IDN_REMOTE_FETCH_CONFIG(
-    ID INTEGER NOT NULL,
+	ID VARCHAR(255) NOT NULL,
```
Comment (jeradrutnam) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql:1536]: Minor indentation issue here

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mysql.sql [hunk 2]
```diff
@@ -1067,8 +1067,8 @@ CREATE TABLE IDN_REMOTE_FETCH_CONFIG (
 )ENGINE INNODB;
 
 CREATE TABLE IDN_REMOTE_FETCH_REVISIONS (
-	ID INT NOT NULL AUTO_INCREMENT,
-	CONFIG_ID INT NOT NULL,
+    ID VARCHAR(255) NOT NULL,
```
Comment (NipuniBhagya) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mysql.sql:1070]: Let's fix this indentation issue

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql (hunk 1 above)
Comment (Thumimku) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql:1536]: Fixed in https://github.com/wso2/carbon-identity-framework/pull/3143

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mysql.sql (hunk 2 above)
Comment (Thumimku) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mysql.sql:1070]: Fixed in https://github.com/wso2/carbon-identity-framework/pull/3143

**Pull request merged**